searx/search/checker/impl.py

```diff
@@ -25,13 +25,87 @@
 logger = logger.getChild('searx.search.checker')
 
 HTML_TAGS = [
-    'embed', 'iframe', 'object', 'param', 'picture', 'source', 'svg', 'math', 'canvas', 'noscript', 'script',
-    'del', 'ins', 'area', 'audio', 'img', 'map', 'track', 'video', 'a', 'abbr', 'b', 'bdi', 'bdo', 'br', 'cite',
-    'code', 'data', 'dfn', 'em', 'i', 'kdb', 'mark', 'q', 'rb', 'rp', 'rt', 'rtc', 'ruby', 's', 'samp', 'small',
-    'span', 'strong', 'sub', 'sup', 'time', 'u', 'var', 'wbr', 'style', 'blockquote', 'dd', 'div', 'dl', 'dt',
-    'figcaption', 'figure', 'hr', 'li', 'ol', 'p', 'pre', 'ul', 'button', 'datalist', 'fieldset', 'form', 'input',
-    'label', 'legend', 'meter', 'optgroup', 'option', 'output', 'progress', 'select', 'textarea', 'applet',
-    'frame', 'frameset'
+    'embed',
+    'iframe',
+    'object',
+    'param',
+    'picture',
+    'source',
+    'svg',
+    'math',
+    'canvas',
+    'noscript',
+    'script',
+    'del',
+    'ins',
+    'area',
+    'audio',
+    'img',
+    'map',
+    'track',
+    'video',
+    'a',
+    'abbr',
+    'b',
+    'bdi',
+    'bdo',
+    'br',
+    'cite',
+    'code',
+    'data',
+    'dfn',
+    'em',
+    'i',
+    'kdb',
+    'mark',
+    'q',
+    'rb',
+    'rp',
+    'rt',
+    'rtc',
+    'ruby',
+    's',
+    'samp',
+    'small',
+    'span',
+    'strong',
+    'sub',
+    'sup',
+    'time',
+    'u',
+    'var',
+    'wbr',
+    'style',
+    'blockquote',
+    'dd',
+    'div',
+    'dl',
+    'dt',
+    'figcaption',
+    'figure',
+    'hr',
+    'li',
+    'ol',
+    'p',
+    'pre',
+    'ul',
+    'button',
+    'datalist',
+    'fieldset',
+    'form',
+    'input',
+    'label',
+    'legend',
+    'meter',
+    'optgroup',
+    'option',
+    'output',
+    'progress',
+    'select',
+    'textarea',
+    'applet',
+    'frame',
+    'frameset',
 ]
```

Comment:
Over all I think black formatting is good, but there are some exceptions like this one here.

In such cases we should use `# fmt: off` & `# fmt: on`.